hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -105,6 +105,7 @@ private void setUpDataDir() {
    * Create Symbolic Link for the HDFS binary.
    */
   private void createSymbolicLink(ExecutorDriver driver) {
+    // todo:  (kgs) file util
```

Comment:
Count me as a vote against in code Tasks.  I'd prefer JIRAs so all issues are centralized in one place.  It also forces a clearer definition of what needs to be done.  What happens if you get hit by a bus Ken?  We will never know how to complete this important task.